.circleci/config.yml

```diff
@@ -43,6 +43,61 @@ jobs:
       # run tests!
       - run: gradle test
 
+  Spinach:
+    docker:
+      # specify the version you desire here
+      - image: circleci/openjdk:8-jdk
+
+      # Specify service dependencies here if necessary
+      # CircleCI maintains a library of pre-built images
+      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # - image: circleci/postgres:9.4
+
+    working_directory: ~/repo
+
+    environment:
+      # Customize the JVM maximum heap limit
+      JVM_OPTS: -Xmx3200m
+      TERM: dumb
+
+    steps:
+      - checkout
+
+      - run: echo ${CIRCLE_BRANCH}
+
+      # Download and cache dependencies
+      - restore_cache:
+          keys:
+            - v1-dependencies-{{ checksum "build.gradle" }}
+            # fallback to using the latest cache if no exact match is found
+            - v1-dependencies-
+
+      - run: gradle dependencies
+
+      - save_cache:
+          paths:
+            - ~/.gradle
+          key: v1-dependencies-{{ checksum "build.gradle" }}
+
+      - run:
+          name: Install spinach dependencies
+          working_directory: ~/repo/http_server_spec
+          command: ../spinach.sh
+
+      - run:
+          name: run server
+          command: gradle server
+          background: true
+
+      - run:
+          name: Run acceptance tests
+          working_directory: ~/repo/http_server_spec
+          command: |
+            sleep 30
```

Comment:
Sleep is usually not the ideal solution. I would recommend you look into a solution with netcat where you wait until the server is up.

Reply:
I saw that solution, which has a timeout associated with it so I figured sleep accomplishes all in one step.  I can modify.